packages/fuel-indexer/src/queries.rs

```diff
@@ -0,0 +1,102 @@
+use cynic::QueryBuilder;
```

Comment:
We should make a tracking issue for making sure that all of this lands in `fuel-core` so we don't have duplicated code that's susceptible to drift. Not a blocker though.

Reply:
@deekerno feel free! You're right copying the code is just a temp solution